airflow/dags/oncokb_curation.py

```diff
@@ -0,0 +1,136 @@
+from datetime import timedelta, datetime
+from airflow.decorators import dag, task
+from airflow.io.path import ObjectStoragePath
+import pandas as pd
+import logging
+
+#Read data locally, we can change it to msk API later
+local = "path/to/folder/with/msk/data"
+base = ObjectStoragePath("s3://my_aws_s3@oncokb-ge")
+filter_path = "path/to/data"
+
+default_args = {
+    	'owner': 'oncoKB',
+		'retries': 3,
+		'retry_delay': timedelta(minutes=5)
+}
+
+@dag(
+    default_args=default_args,
+    dag_id="oncokb_curation_v1",
+    description="Get the statistic file and save to S3",
+    start_date=datetime(2024,6,8,0),
+    schedule_interval="@weekly",
+    catchup=False,
+)
+
+def curate_mutation_data():
+
+    def exclude_samples(data,filter_path):
+        excluded_sample_ids = pd.read_csv(filter_path,delimiter='\t',header=None)[0].tolist()
+        filtered_data = data[~data['SAMPLE_ID'].isin(excluded_sample_ids)]
+        return filtered_data
+    
+    def upload_to_s3(path,df):
```

Comment:
How is airflow sending this file to s3? It looks like you have local paths being referenced in the code.